pyproject.toml

```
@@ -117,46 +161,19 @@ include = ["rsciio*"]
 fallback_version = "0.3.dev0"
 
 [tool.towncrier]
-package = "rsciio"
-filename = "CHANGES.rst"
 directory = "upcoming_changes/"
-title_format = "{version} ({project_date})"
+filename = "CHANGES.rst"
 issue_format = "`#{issue} <https://github.com/hyperspy/rosettasciio/issues/{issue}>`_"
-
-  [[tool.towncrier.type]]
-  directory = "new"
-  name = "New features"
-  showcontent = true
-
-  [[tool.towncrier.type]]
-  directory = "bugfix"
-  name = "Bug Fixes"
-  showcontent = true
-
-  [[tool.towncrier.type]]
-  directory = "doc"
-  name = "Improved Documentation"
-  showcontent = true
-
-  [[tool.towncrier.type]]
-  directory = "deprecation"
-  name = "Deprecations"
-  showcontent = true
-
-  [[tool.towncrier.type]]
-  directory = "enhancements"
-  name = "Enhancements"
-  showcontent = true
-
-  [[tool.towncrier.type]]
-  directory = "api"
-  name = "API changes"
-  showcontent = true
-
-  [[tool.towncrier.type]]
-  directory = "maintenance"
-  name = "Maintenance"
-  showcontent = true
+package = "rsciio"
```

```suggestion
title_format = "{version} ({project_date})"
package = "rsciio"
```
otherwise, the title is as
```
Rsciio 0.3 (2023-12-12)
=======================
```
instead of 
```
0.3 (2023-12-12)
================
```